server/services/interventionsService.test.ts

```diff
@@ -279,6 +279,189 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
         '5352cfb6-c9ee-468c-b539-434a3e9b506e',
       ])
     })
+
+    it('returns the updated referral when setting additionalNeedsInformation and accessibilityNeeds', async () => {
+      await provider.addInteraction({
+        state: 'a draft referral with ID dfb64747-f658-40e0-a827-87b4b0bdcfed exists',
+        uponReceiving: 'a PATCH request to set additionalNeedsInformation and accessibilityNeeds',
+        withRequest: {
+          method: 'PATCH',
+          path: '/draft-referral/dfb64747-f658-40e0-a827-87b4b0bdcfed',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            Authorization: `Bearer ${token}`,
+          },
+          body: {
+            accessibilityNeeds: 'She uses a wheelchair',
+            additionalNeedsInformation: 'Alex is currently sleeping on her aunt’s sofa',
+          },
+        },
+        willRespondWith: {
+          status: 200,
+          body: {
+            id: Matchers.like('dfb64747-f658-40e0-a827-87b4b0bdcfed'),
```

Comment:
i think this response would contain all the other fields on the referral, whether null or otherwise, like 

```{
    "id": "97385a77-0cec-4ee0-a2a5-dcad0ecf7101",
    "created": "2021-01-04T13:08:59.329395Z",
    "completionDeadline": null,
    "serviceCategoryId": null,
    "complexityLevelId": null,
    etc....
}```

Reply:
You're right, I think it would. But this approach (of only testing for the updated fields) is consistent with the other `PATCH` contract tests in this file. If the response contained additional fields, then the contract tests would still pass. I think that what we're testing for with these contract tests is that the response to a `PATCH` request correctly reflects the data which was submitted in this request.

There's a broader question which is whether we should be asserting anything about these `PATCH` response bodies at all, given that the UI doesn't actually make use of them at the moment.